tests/end2end/data/hints/ace/ace.js

```diff
@@ -8182,7 +8182,7 @@ function Folding() {
 
     this.foldAll = function(startRow, endRow, depth) {
         if (depth == undefined)
-            depth = 100000; // JSON.stringify doesn't hanle Infinity
+            depth = 100000; // JSON.stringify doesn't handle Infinity
```

Comment:
This is in bundled third-party code (ace.js) - I'd prefer to keep it identical to the upstream copy.

Reply:
same here

Reply:
This is also still open.